docs/content/dependencies/deep-storage.md

```diff
@@ -21,24 +23,12 @@ If you are using the Hadoop indexer in local mode, then just give it a local fil
 
 ## S3-compatible
 
-S3-compatible deep storage is basically either S3 or something like Google Storage which exposes the same API as S3.
-
-S3 configuration parameters are
-
-|Property|Possible Values|Description|Default|
-|--------|---------------|-----------|-------|
-|`druid.s3.accessKey`||S3 access key.|Must be set.|
-|`druid.s3.secretKey`||S3 secret key.|Must be set.|
-|`druid.storage.bucket`||Bucket to store in.|Must be set.|
-|`druid.storage.baseKey`||Base key prefix to use, i.e. what directory.|Must be set.|
+See [here](../development/extensions-core/s3.html).
```

Comment:
"See the [druid-s3-extensions extension documentation]." would read better than "See [here]."